aws_lambda_powertools/event_handler/util.py

```diff
@@ -11,3 +16,24 @@ class _FrozenDict(dict):
 
     def __hash__(self):
         return hash(frozenset(self.keys()))
+
+
+def extract_origin_header(resolver_headers: Dict[str, Any]):
+    """
+    Extracts the 'origin' or 'Origin' header from the provided resolver headers.
+
+    The 'origin' or 'Origin' header can be either a single header or a multi-header.
+
+    Args:
+        resolver_headers (Dict): A dictionary containing the headers.
+
+    Returns:
+        Optional[str]: The value(s) of the origin header or None.
+    """
+    resolved_header = get_header_value(resolver_headers, "origin", None, case_sensitive=False)
+    if isinstance(resolved_header, str):
+        return resolved_header
+    if isinstance(resolved_header, list):
+        return resolved_header[0]
+
+    return resolved_header
```

Comment:
```suggestion
    if isinstance(resolved_header, list):
        return resolved_header[0]

    return resolved_header
```

Did you mean like this? Since you're always returning at the end

Reply:
Great point!